crates/gitbutler-project/src/project.rs

```diff
@@ -96,6 +96,18 @@ pub struct Project {
     pub snapshot_lines_threshold: Option<usize>,
     #[serde(default = "default_false")]
     pub succeeding_rebases: bool,
+    #[serde(default)]
+    pub git_host: GitHostSettings,
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone, Default)]
+pub struct GitHostSettings {
+    #[serde(default)]
+    pub host_type: Option<String>,
+    #[serde(default)]
+    pub use_pull_request_template: Option<bool>,
+    #[serde(default)]
+    pub pull_request_template_path: Option<String>,
```

Comment:
Could you put a `#[serde(rename_all = "camelCase")]` right after the `#[derive(...)]`?

Reply:
Brilliant, applied :+1: